priceoyeclone/urls.py

```diff
@@ -1,7 +1,12 @@
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
 from django.urls import include, path
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("Users/", include("Users.urls")),
+    path("users/", include("users.urls")),
+    path('password_reset/', auth_views.PasswordResetView.as_view(), name='password_reset'),
```

Comment:
according to URL format guidelines URL should not use `_` because this make URL less readable like password_reset/ instead you can use `-` in place of `_`